### ISSUE TITLE
Some mild updates required to the style page on Breadcrumbs

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon-website#2821]
On the [Style page](https://v11.carbondesignsystem.com/components/breadcrumb/style/), some of the information may need attention.

Under Color, there is a series of images showing state appearance
![image](https://user-images.githubusercontent.com/14298245/160489875-84e74ca2-539c-479d-a544-da5873c3cbed.png)

- [x] The Focus state should have both a focus indicator _and_ a blue underline
- [x] I don't think the Active image should exist. isn't the "Current page" the same as active? And I can't think of a situation where you'd have an underlined non-interactive element.

----

Under Truncated breadcrumbs, the page states:
> Truncated breadcrumbs are not currently built into the breadcrumb component

It's unclear if this means that a page link (i.e., page name that is a breadcrumb link) is never truncated. There are no illustrations. This sentence is then followed by  a comment on the overflow menu, which can either be interpreted as a further puzzling comment on the first sentence, or a related topic about where not all breadcrumbs are shown (truncating the number of links listed, as opposed to truncating a portion of a single link name). The guidance needs to be clarified.

Under the Structure section, there is an image of an expanded breadcrumb overflow menu, which lacks a focus indicator on the first item (Breadcrumb 2) in the expanded list. This is the same issue (but different page) identified in https://github.com/carbon-design-system/carbon-website/issues/2815
![image](https://user-images.githubusercontent.com/14298245/160491653-d362dfdc-019d-452e-a7c9-e3ac91326620.png)

**Comment by laurenmrice** [comment]
### Update 04/25

**Active state**
> I don't think the Active image should exist. isn't the "Current page" the same as active? And I can't think of a situation where you'd have an underlined non-interactive element.

The Active state is when you click/press on the link, it turns a different color in that state. We have updated it to this below:

<img width="330" alt="Image" src="https://github.com/user-attachments/assets/1fd03d25-fb85-470e-8f1d-62e5f7f8cb98" />

----

**Truncated breadcrumbs discussion**
We have updated this section to refer to this as overflow and not truncation. The first item in the menu now has a focus state.

<img width="494" alt="Image" src="https://github.com/user-attachments/assets/15b0f80f-26ab-4ebf-81c4-9a28d82bb978" />

**Comment by laurenmrice** [comment]
Closing because I believe this has been resolved now.
